src/OrchardCore.Modules/OrchardCore.Menu/Models/ContentPickerMenuItemPart.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.Routing;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Routing;
+
+namespace OrchardCore.Menu.Models
+{
+    public class ContentPickerMenuItemPart : ContentPart
+    {
+        /// <summary>
+        /// The name of the link
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Content Picker ItemIds
+        /// </summary>
+        public string ContentItemIds { get; set; }
+
```

Comment:
This should be `ContentItemId` because the picker does not allow multiples I think

Reply:
It allows multiple

Reply:
I think multiple selecting is necessary,it should be possible a menu item contains three contenttype contents, this may take more work, so I temporarily changed it to ContentItemId.

Reply:
Yes, the second option @Skrypt 

What I meant here, is that because the menu text is singular, you wouldn't want to be selecting multiple content items, because you would only have one name to render them with.

So for me, in this context, you should only ever be able to pick a single content item per menu item.

And then if you want to add multiple items to a menu, you would add multiple menuitems, with a single contentitems picked from each.

Reply:
> And then if you want to add multiple items to a menu, you would add multiple menuitems, with a single contentitems picked from each.

That's wonderful.